### DISCUSSION TITLE
Ubuntu 22.04 LST problem

### DISCUSSION THREAD

**Discussion by meineko3** [phoebe-project/phoebe2#634] [Questions]
Recently I upgraded OS from Ubuntu 20.04 LST to 22.04 LST
After the upgrade, I have a problem running phoebe 2.4.4.

For example, phoebe might work well, but the graph did not appear in the following test script. 

>import phoebe
>b=phoebe.default_binary()
>b.add_dataset('lc', times=phoebe.linspace(0,2,101))
>b.run_compute()
>b.plot(x='phases', show=True)

The error message was as follows;

>Traceback (most recent call last):
 > File "/home/meineko/test.py", line 5, in <module>
    b.plot(x='phases', show=True)
  >File "/home/meineko/.local/lib/python3.10/site-packages/phoebe/parameters/parameters.py", line 5955, in plot
    return self._show_or_save(save, show, animate,
 > File "/home/meineko/.local/lib/python3.10/site-packages/phoebe/parameters/parameters.py", line 6073, in _show_or_save
    fig = afig.draw(i=time,
  >File "/home/meineko/.local/lib/python3.10/site-packages/phoebe/dependencies/autofig/figure.py", line 585, in draw
    fig = self._get_backend_object(fig, naxes=len(self.axes))
  >File "/home/meineko/.local/lib/python3.10/site-packages/phoebe/dependencies/autofig/figure.py", line 284, in _get_backend_object
    fig.set_figwidth(8*cols)
 >File "/home/meineko/.local/lib/python3.10/site-packages/matplotlib/figure.py", line 3021, in set_figwidth
    self.set_size_inches(val, self.get_figheight(), forward=forward)
 > File "/home/meineko/.local/lib/python3.10/site-packages/matplotlib/figure.py", line 2959, in set_size_inches
    manager.resize(*(size * self.dpi).astype(int))
  >File "/home/meineko/.local/lib/python3.10/site-packages/matplotlib/backends/_backend_gtk.py", line 236, in resize
    toolbar_size = self.toolbar.size_request()
>AttributeError: 'NavigationToolbar2GTK4' object has no attribute 'size_request'. Did you mean: 'get_size_request'?

What's wrong?
Phoebe worked well in Ubuntu 20.04 LST
Any advice? I appreciate it.

Seiichiro Kiyota
VSOLJ, Japan

**Comment by kecnry** [comment]
Can you check your version of matplotlib (`import matplotlib, print(matplotlib.__version__)`?  We've had some reports of issues with 3.5.2, although nothing that caused the default options to fail.  If that is the culprit, downgrading to 3.5.1 should fix all the plotting issues, and I'll file and issue with all the various compatibility problems and try to bump that up on the priority list.

Thanks!
-Kyle

**Reply by meineko3** [reply]
Thank you, Kyle

Version of matplotlib is 3.6.0

Seiichiro Kiyota

**Reply by kecnry** [reply]
Ok, please downgrade your matplotlib (or perhaps try switching the backend to something other than GTK) and let me know if that does or doesn't fix the issue.  I filed an issue to try to support newer version of matplotlib as #636.

**Reply by meineko3** [reply]
Thank you

I downgrade matplotlib from 3.6.0 to 3.5.0.
I can show the graph in phoebe.

Seiichiro Kiyota
